bigquery/setup.py

```diff
@@ -30,7 +30,7 @@
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
     'enum34; python_version < "3.4"',
-    "google-cloud-core >= 1.0.0, < 2.0dev",
+    "google-cloud-core >= 1.0.3, < 2.0dev",
```

Comment:
Maybe I missed the comment, but why are we bumping the minimum version?

Reply:
I saw this change in the already merged PR (Translate: Add client options to translate_v2.), which I was using as an example.

Reply:
@busunkim96 Is this dependency update required?

Reply:
Yes, `google-cloud-core` 1.0.3 bumps the minimum `google-api-core` version to 1.14.0 (`client_options` was added in 1.13.0). See [core/CHANGELOG.md](https://github.com/googleapis/google-cloud-python/blob/master/core/CHANGELOG.md), [api_core/CHANGELOG.md](https://github.com/googleapis/google-cloud-python/blob/master/api_core/CHANGELOG.md)